src/build/__tests__/build-test.js

```diff
@@ -125,8 +144,8 @@ describe('build()', () => {
 
   it('should throw if building a schema and types is not an array', () => {
     const msg = 'Can\'t use thunks as type dependencies for schema.';
-    expect(() => build(schemaSource + querySource, undefined, () => {})).toThrowError(msg);
-    expect(() => build(querySource, undefined, () => {})).toThrowError(msg);
+    expect(() => build(schemaSource + querySource, undefined, () => { })).toThrowError(msg);
```

Comment:
Looks like the editor added a space between the brackets. Can we get it to what it was before?